src/Provider/ProductUrlProvider.php

```diff
@@ -33,13 +33,13 @@ final class ProductUrlProvider implements UrlProviderInterface
 
     private ChannelInterface $channel;
 
-    /** @var string[] */
-    private ?array $channelLocaleCodes;
```

Comment:
Not sure we should change this, it's a bit indifferent to me although, let's try to keep changes as much related to the skeleton update :)